apps/api/v2/src/modules/oauth-clients/oauth-client.repository.ts

```diff
@@ -95,8 +95,21 @@ export class OAuthClientRepository {
   }
 
   async deleteOAuthClient(clientId: string): Promise<PlatformOAuthClient> {
-    return this.dbWrite.prisma.platformOAuthClient.delete({
-      where: { id: clientId },
+    return await this.dbWrite.prisma.$transaction(async (prisma) => {
```

Comment:
This is a scary query. Can we please add automated tests to ensure it only deletes the users we want it to delete?

Reply:
Why do we want to delete the users?